pkg/term/windows/ansi_reader.go

```diff
@@ -10,8 +10,8 @@ import (
 	"strings"
 	"unsafe"
 
-	. "github.com/Azure/go-ansiterm"
-	. "github.com/Azure/go-ansiterm/winterm"
+	ansiterm "github.com/Azure/go-ansiterm"
```

Comment:
@vdemeester if golint is not complaining i would keep it not aliased `import "github.com/Azure/go-ansiterm"`

@jstarks what matters is the package name (in this case: `package ansiterm`) not the import path.